net/openthread-br/Makefile

```diff
@@ -94,10 +96,8 @@ endef
 
 define Package/openthread-br/install
 	$(INSTALL_DIR) \
-		$(1)/etc/init.d \
```

Comment:
Why?

Reply:
Right. Forgot I moved things to a netifd proto handler. This one can be removed indeed.